.ci/no-exception-test.sh

```diff
@@ -59,7 +63,7 @@ openjdk14-with-checks-nonjavadoc-error)
       --mode single \
       --patchConfig checks-nonjavadoc-error.xml \
       --localGitRepo  "$LOCAL_GIT_REPO" \
-      --patchBranch "$BRANCH"
+      --patchBranch "$BRANCH" -xm "Dcheckstyle.failsOnError=false"
```

Comment:
please keep `-` in front of `D` to make it clear that it is just argument to transfer CLI parameters to inner mvn.

Reply:
Done.